src/Forms/FieldGroup.php

```diff
@@ -154,7 +154,7 @@ public function getMessage()
         /** @var FormField $subfield */
         $messages = [];
         foreach ($dataFields as $subfield) {
-            $message = $subfield->obj('Message')->forTemplate();
+            $message = $subfield->obj('Message')?->forTemplate();
```

Comment:
`obj()` and `dbObject()` can return `null`, so I've added a bunch of these type safety nets.